pom.xml

```diff
@@ -205,6 +205,12 @@
             <version>1.2.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Is this actually needed. Can't we use the test framework used by grizzly-ahc so that we maintain coherence?